### DISCUSSION TITLE
AccessKit alt text: Make user-selectable(?)

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#519] [Suggestion Box]
Not sure if this is a) actually a good idea or b) easy to do without non-semantic or hacky DOM manipulation, but I sometimes find myself trying to select the text in an AccessKit caption, which doesn't work because it's part of the button that opens the lightbox. If other people also want this, maybe it would be worth looking into.

**Comment by AprilSylph** [comment]
It would probably be possible (and easy) to just slap an `event => event.stopPropagation()` click handler on each caption. I've actually run into the same problem myself.

**Comment by marcustyphoon** [comment]
This was implemented in #520.
